html/semantics/embedded-content/the-img-element/update-the-image-data/element-delay-the-load-event.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Inline image element blocks load</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<img src="/images/blue.png?pipe=trickle(d7)">
+<script>
+var img_loaded = false;
+
+var f = function() {
+  img_loaded = true;
+};
```

Comment:
Why don't you make this simpler by putting the `<script>` before the `<img>` in the markup?  You can add the load handler to the `<img>` with an onload attribute instead of a script to avoid the (hopefully completely theoretical) possibility of the image loading before the script can add the listener.